### DISCUSSION TITLE
closeLottery() is failed in Lottery on Goerli Testnet

### DISCUSSION THREAD

**Discussion by 0xlightfury** [community/community#40759] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I forked pancake's lottery on goerli testnet.
But while I tried to close lottery, the transaction was failed.
Here is lottery contract.
0xf719C03B28824126ac92e785f6317d3860a38E6a
And here is failed transaction.
0x8a915455e60679e9b5ea13a3272a36e72fefac1fc63ef7af00417875a9601c70
Please let me know what the reason is.
Thanks.

**Comment by jfullstackdev** [comment]
you mean this repo ? 

https://github.com/pancakeswap/pancake-lottery-scheduler

you can ask that same question by raising an issue at that repo

**Reply by 0xlightfury** [reply]
Thank you for your attention.
It is contract's issue.
I hope help in smart contract's side such as parameters' setting.
